### PR TITLE
test: use eslint in daily tests

### PR DESCRIPTION
**Pull request by sdbondi**
Description
---
- adds eslint to daily tests
- fix a number of lints
- fix broken git pull command

Motivation and Context
---
Small feedback loop for errors on daily tests will prevent constant fixing of "silly" mistakes 

How Has This Been Tested?
---
eslint passes

### CODE REVIEW

File: applications/daily_tests/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["prettier", "@babel"],
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "node": true,
+    "es2021": true
+  },
+  "parser": "@babel/eslint-parser", // Tell ESLint that you want to use the @babel/eslint-parser.
```
Comment (delta1) [inline applications/daily_tests/.eslintrc.json:10]: lol I thought comments weren't allowed in json

**Review by delta1 — APPROVED** [review]
LGTM untested

File: applications/daily_tests/.eslintrc.json (hunk 1 above)
Comment (delta1) [reply applications/daily_tests/.eslintrc.json:10]: interesting https://github.com/mozilla/addons-frontend/issues/942

**Review by stringhandler — APPROVED** [review]

File: applications/daily_tests/.eslintrc.json (hunk 1 above)
Comment (sdbondi) [reply applications/daily_tests/.eslintrc.json:10]: Yeah also saw that, very needed - basically the worst thing about JSON as config

**Pull request merged**